src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces/doc_viewer_span_overview/hooks/use_transaction.tsx

```diff
@@ -23,35 +23,44 @@ interface GetTransactionParams {
   transactionId: string;
   indexPattern: string;
   data: DataPublicPluginStart;
+  signal: AbortSignal;
 }
 
-async function getTransactionData({ transactionId, indexPattern, data }: GetTransactionParams) {
+async function getTransactionData({
+  transactionId,
+  indexPattern,
+  data,
+  signal,
+}: GetTransactionParams) {
   return lastValueFrom(
-    data.search.search({
-      params: {
-        index: indexPattern,
-        size: 1,
-        body: {
-          timeout: '20s',
-          query: {
-            bool: {
-              must: [
-                {
-                  term: {
-                    'transaction.id': transactionId,
+    data.search.search(
+      {
+        params: {
+          index: indexPattern,
+          size: 1,
+          body: {
+            timeout: '20s',
+            query: {
+              bool: {
+                must: [
+                  {
+                    term: {
+                      'transaction.id': transactionId,
+                    },
                   },
-                },
-                {
-                  term: {
-                    'processor.event': 'transaction',
+                  {
+                    term: {
+                      'processor.event': 'transaction',
```

Comment:
```suggestion
                      [PROCESSOR_EVENT_FIELD]: 'transaction',
```

Maybe also replace `transaction` with an enum/constant

Reply:
https://github.com/elastic/kibana/commit/c4bed777ab744a3e87080b10fdf214364e63b09e